spec/components/filters/exclude_future_dated_spec.rb

```diff
@@ -0,0 +1,38 @@
+require 'spec_helper'
+require 'filters/exclude_future_dated'
+
+RSpec.describe Filters::ExcludeFutureDated do
+  subject { described_class.new('datetime').call }
+
+  context 'when a records datetime field is in the future' do
+    let(:record) { { 'datetime' => 1.hour.from_now.to_s, 'id' => 123 } }
```

Comment:
The only thing that is changing between each context here is the `datetime` value.

I would define `record` earlier on, then within each context call out how the date time has changed:

```rb
let(:record) { { 'datetime' => datetime.to_s, 'id' => double } }

context 'when a records datetime field is in the future' do
  let(:datetime) { 1.hour.from_now }
  # ...
end

context 'when a records datetime field is in the past (yesterday)' do
  let(:datetime) { 1.day.ago }
  # ...
end
```